perf/lua/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_custom_target(test-lua-perf
+                  DEPENDS 1mops_write_perftest
+                          box_select_perftest
+                          uri_escape_unescape_perftest
+                  COMMENT "Running Lua performance tests"
+)
+
+set(TARANTOOL_BIN ${PROJECT_BINARY_DIR}/src/tarantool)
+
+add_custom_target(1mops_write_perftest
```

Comment:
Too much copy-pasting IMHO. I suggest to create a CMake macro/function to ease further benchmarks additions.